### PR TITLE
clickbits.clockodo version 6.0.10

### PR DESCRIPTION
**Pull request by Keller0Junge**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16541)

### CODE REVIEW

File: manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: clickbitsGmbH.clockodo
```
Comment (ItzLevvie) [inline manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.yaml:3]:
```suggestion
PackageIdentifier: clickbits.clockodo
```

File: manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: clickbitsGmbH.clockodo
```
Comment (ItzLevvie) [inline manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.locale.de-DE.yaml:3]:
```suggestion
PackageIdentifier: clickbits.clockodo
```

File: manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: clickbitsGmbH.clockodo
```
Comment (ItzLevvie) [inline manifests/c/clickbitsGmbH/clockodo/6.0.10/clickbitsGmbH.clockodo.installer.yaml:3]:
```suggestion
PackageIdentifier: clickbits.clockodo
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

Just make sure to rename the folder names using the GitHub Desktop client on Windows.

**Comment by Keller0Junge** [comment]
@ItzLevvie i had some trouble to change the path. I am pretty new in Git. It was much easier for me to creat a new request with these changes (number 16623). Iwill try to learn an understand how to change my request and path in time but for now it should be fine with a new request.

**Pull request closed without merging**